src/geotop/water.balance.cc

```diff
@@ -1736,7 +1736,6 @@ int find_f_3D(double Dt, Vector<double> *f, ALLDATA *adt, SOIL_STATE *L,
     long i;
     long n=(Nl+1)*adt->P->total_pixel;
 
-    #pragma omp parallel for
```

Comment:
@ElisaBortoli,
I took a second look of this function and I think the pragma is correct. Do you mind to point out which lines are not thread safe?

Reply:
I would have put some variables private (i.e, ```i, l``` etc) since some calculations depend on them.
Then  @line 1758  ```area -= (*adt->C->length)(ch) * adt->P->w_dx * ds; //area of the pixel[m2]
```, @line ```area = (*adt->C->length)(ch) * adt->P->w_dx * ds;``` so there is a reduction but only in certain cases and also for ```f->co[i]```, so before putting a ```#pragma omp parallel for``` I wanted to check all what's written inside.

Why do you say that neither reductions nor private variables are needed?

I just want to be sure to not introduce mistakes in the code.

Reply:
`i` is private by default because it is the counter in the for loop. `area` is private because declared inside the parallel region